webui/src/pages/setup/index.jsx

```diff
@@ -108,4 +126,9 @@ server:
     );
 };
 
+const SetupPage = () => <>
+			    <SetupWarnings/>
```

Comment:
I think it looks a little out of place. At a minimum it should be on the same grid as the setup form (above it, below it, idc)

Reply:
On the "upload object" modal on the objects page:
!["upload object" modal with warning about using local block adapter](https://user-images.githubusercontent.com/7802932/120922026-8feaed00-c6cf-11eb-929c-cc82cb74c05d.png)